### DISCUSSION TITLE
Uploading folders like Cyberduck

### DISCUSSION THREAD

**Discussion by frozenfroze** [isontheline/pro.webssh.net#852] [Ideas]
The current implementation of SFTP upload files and folder is very lacking.

I will like to be able to drag and drop my folder into the SFTP or choose the folder to update to immediately upload to the current folder with the ability to track the progress (retry on failure), selections for choosing to override or skip or rename older file when same filename exist.

Right now I can't use the SFTP as I always need to upload the whole folder and it's impossible to use as it only allows uploading 1 file per time

**Comment by isontheline** [comment]
@frozenfroze Yes WebSSH SFTP implementation lacks a lot of useful features inside Cyberduck / FileZilla.

I will do my best to improve it.

**Comment by xerosys** [comment]
The ability to upload a whole folder is much needed.  Drag and drop would be amazing, but not necessary as long as there's a menu option available for it.  Thank you for considering to add this feature.

**Reply by isontheline** [reply]
Thanks for your feedback @xerosys 🙏

I will drive up these issues :
* https://github.com/isontheline/pro.webssh.net/issues/490
* https://github.com/isontheline/pro.webssh.net/issues/205
